domains/Discussions/Controllers/AnswersUpdateController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Answer;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AnswersUpdateController extends Controller
+{
+    private Answer $answer;
+    private Question $question;
+
+    public function __construct(Question $question, Answer $answer)
+    {
+        $this->question = $question;
+        $this->answer = $answer;
+    }
+
+    public function __invoke(Request $request, int $questionId, int $answerId): Response
+    {
+        $answer = $this->answer->OfQuestion($questionId)->findOrFail($answerId);
```

Comment:
The scope name must be camelCased. Also, with that scope, there's no need to search for the question, specifically.